workers/vita/buildbot.cfg

```diff
@@ -1,6 +1,6 @@
 builders = {
     "": {
-        "configure_args": ["--host=psp2"],
+        "configure_args": ["--host=psp2 --with-freetype2-prefix=$VITASDK/arm-vita-eabi/ --enable-freetype2"],
```

Comment:
1. This is an arguments array. It is not a string. It is not correct to multiple flags in one argument like this.
2. The build system automatically detects and configures libraries. Passing explicit flags to enable features like this is inappropriate.
3. The worker image should be configured so that the build system’s auto-configuration will find Freetype. Passing its prefix explicitly in this manner is inappropriate.